Dockerfile

```diff
@@ -41,20 +43,20 @@ RUN /setup.sh
 # Configure localhost in Apache
 RUN cp  /tmp/resources/000-default.conf /etc/apache2/sites-available/
 
-# To be able to install libapache.
-RUN echo 'deb http://archive.ubuntu.com/ubuntu trusty multiverse' >> /etc/apt/sources.list
-RUN echo 'deb http://archive.ubuntu.com/ubuntu trusty-updates multiverse' >> /etc/apt/sources.list
-RUN echo 'deb http://security.ubuntu.com/ubuntu trusty-security multiverse' >> /etc/apt/sources.list
-RUN  apt-get update
+# To be able to install libapache. #Allready in source.list for ubuntu:focal
+#RUN echo 'deb http://archive.ubuntu.com/ubuntu focal multiverse' >> /etc/apt/sources.list
+#RUN echo 'deb http://archive.ubuntu.com/ubuntu focal-updates multiverse' >> /etc/apt/sources.list
+#RUN echo 'deb http://security.ubuntu.com/ubuntu focal-security multiverse' >> /etc/apt/sources.list
+#RUN  apt-get update
 
-# Install PHP5 and necessary modules
-RUN  apt-get install -y libapache2-mod-fastcgi php5-fpm libapache2-mod-php5 php5-common php5-cli php5-fpm php5
+# Install PHP7.4 and necessary modules
+RUN  apt-get install -y php7.4-fpm libapache2-mod-php7.4 php7.4-common php7.4-cli php7.4-fpm php7.4 #libapache2-mod-fastcgi doesn't exist anymore
 
 # Enable these Apache modules
 RUN  a2enmod actions cgi alias
 
-# Apache configuration for PHP-FPM
-RUN cp /tmp/resources/php5-fpm.conf /etc/apache2/conf-available/
+# Apache configuration for PHP-FPM # No fastcgi anymore
```

Comment:
How about the following https://www.interserver.net/tips/kb/apache-php-fpm-ubuntu-18-04/

Reply:
Yes, I've tested it and it's working for me.
I forgot to comment line 57 cp php5-fpm.conf
Focal is 20-04 and I haven't found apache2 fastcgi in packages.